# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Fri Jan 27 20:35:34 2023 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Fri Jan 27 22:54:08 2023 +0200
@@ -1791,7 +1791,6 @@
       "alpha_y_bpc_mismatch",
       "ispe_mismatch",
       "invalid_cicp",
-      "no_samples",
       "invalid_parse_status",
       "missing_brand",
       "ftyp_not_first",
@@ -1806,8 +1805,7 @@
       "item_type_missing",
       "construction_method",
       "item_loc_not_found",
-      "no_item_data_box",
-      "uncategorized"
+      "no_item_data_box"
     ],
     "description": "Decode result of AVIF image",
     "bug_numbers": [1670827]